project-demo.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+
+<title>Personal Project</title>
+<link rel="stylesheet" href="/Users/khanmohammed1/Documents/Ravine Projects/index.css" type="text/css">
```

Comment:
Since github will not have the same repository you need to access your css file locally like so 
`<link rel="stylesheet" href="./index.css" type="text/css">`
or some variant using ../ or ./ . Remember ../ means to go out one directory and ./ means the same directory

Reply:
Lmk if that makes sense